src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Contracts/CopyResultsRequest.cs

```diff
@@ -22,21 +22,6 @@ public class TableSelectionRange
     /// </summary>
     public class CopyResultsRequestParams : SubsetParams
     {
-        /// <summary>
```

Comment:
@alanrenmsft FYI I'm updating this part and switching to read from config for these 3 settings. I've tested all 3 settings and they are working as expected.